README.md

```diff
@@ -398,9 +523,13 @@ If a user already has manually downloaded a file like a large AI model, should t
 
 ## Considered alternatives
 
+### Adding a description for each file apart from the hash
+
+Apps could reference to the same file identified by a unique hash using different descriptions. For example, an English site could refer to the [`g-2b-it-gpu-int4.bin`](https://storage.googleapis.com/jmstore/kaggleweb/grader/g-2b-it-gpu-int4.bin) AI model as "Gemma AI model from Google", whereas another Spanish site could refer to it as "modelo de IA grande de Google". This information could in theory even be surfaced in the permission prompt. In practice, though, we fear attacks where sites set descriptions like "Click 'Accept' to prove that you are not a bot", which should never make it into a permission prompt. Instead, we envision user agents to enrich permission prompts based on the hashes. For example, a user agent could know that a file identified by a given hash is a well-known AI model and optionally surface this information to the user in the permission prompt.
```

Comment:
Yes!

Reply:
Addressed in https://github.com/tomayac/cross-origin-storage/pull/30/commits/8da7fd34d9747993cc1e74844567e0fafd4c0d8c.